_posts/2021-06-06-announcing_trino_summit.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title:  "Announcing Trino Summit"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+---
+
+Greetings Trino nation,
+
+As mentioned in [last year’s Presto Summit blog](https://blog.starburst.io/presto-summit-sf-postponed-to-2021), 
+“It’s not goodbye, it’s see you later!” We may not have guessed at the time that
+“later” was a year and a half, but here we are. 
+
+Get ready for this year’s hybrid Trino Summit event! This year’s Summit feels a 
+little different coming back for many reasons. For one, the name of the event 
+has changed from Presto to Trino, which means this will be the first event of 
+the project hosted [under the new banner of Trino](https://trino.io/blog/2020/12/27/announcing-trino.html).
+
+<!--more-->
+
+New names and cute bunnies aside, this event has a lot of extra excitement as we
+ring in the first hybrid in-person and virtual Trino event held since the 
+beginning of the pandemic! We're really looking forward to talking in front of a
+live audience with the high fives, hugs, and handshakes we’ve all missed for 
+over a year.
+
+This year's Summit is hosted by Starburst at [the Commonwealth Club](https://www.commonwealthclub.org/visitorinfo)
+in San Francisco, CA on October 13th. We'd originally set the date on 
+September 15th but later realized that this was conflicting with Yom Kippur. 
+While making these updates, it became clear that we could include more people 
+with a hybrid format. If you've already registered, we'll reach out to you with 
+updated information and attendance options. If you haven’t registered yet, 
+[register here](http://starburst.io/trinosummit2021). If you plan to attend in 
+person, for safety purposes, you need to upload a copy of your vaccine card. If 
+you haven’t received your vaccine but plan to receive your final dose two weeks 
+before the date of the conference then you can send us an email at 
+[events@starburst.io](mailto:events@starburst.io).
```

Comment:
Would add a line break before "If you haven't registered yet" to break this paragraph up and draw attention to the action item.

Reply:
done